docparse/processing_options.mdx

```diff
@@ -36,8 +36,13 @@ All of the available options are listed below, and are optional unless specified
     - `merge_across_pages`: A `boolean` that when `True` the selected chunker will attempt to merge chunks across page boundaries. Defaults to `True`. 
 - `output_format`: A string controlling the output representation. Defaults to `json` which yields an array called `elements` which contains the partitioned elements, represented in JSON.  If set to `markdown` the service response will instead include a field called `markdown` that contains a string representing the entire document in Markdown format.
 - `pages_per_call`: This is only available when using the Partition function in Sycamore. This option divides the processing of your document into batches of pages, and you specify the size of each batch (number of pages). This is useful when running OCR on large documents.
+- `output_label_options`: A dictionary of options to specify which heuristic to apply to enforce certain label outputs.
+    - `promote_title`: A boolean that specifies whether to promote an element to title if there's no title in the output.
+    - `title_candidate_elements`:  A list of strings that are candidate elements to be promoted to title. 
```

Comment:
Can you update this to better match the [corresponding docstring](https://github.com/aryn-ai/sycamore/blob/680cea76c4505cdff141256c245101da86fb9a3c/lib/sycamore/sycamore/transforms/partition.py#L422-L430) so that it is more precise? Can you also make it mention what the default is?

```
output_label_options: A dictionary for configuring output label behavior. It supports two options:
    promote_title, a boolean specifying whether to pick the largest element by font size on the first page
        from among the elements on that page that have one of the types specified in title_candidate_elements
        and promote it to type "Title" if there is no element on the first page of type "Title" already.
    title_candidate_elements, a list of strings representing the label types allowed to be promoted to
        a title.
    Here is an example set of output label options:
        {"promote_title": True, "title_candidate_elements": ["Section-header", "Caption"]}
    default: None (no element is promoted to "Title")
```

Reply:
The second sentence in the docs string seems poorly constructed? There's a colon implying a list of options but then a period between the two. We may want to fix the docstring.

I'll add the defaults and the precision as you suggested though

Reply:
I also think 'default: None (no element is promoted to "Title")' in the docstring is a bit misleading. If it applies to the entire option it should just say no heuristic is applied. Right now its just talking about the promote_title heuristic.

Reply:
True.